.eslintrc.json

```diff
@@ -18,7 +18,7 @@
   ],
   "rules": {
     "curly": ["error", "multi-line", "consistent"],
-    "no-console": ["off", { "allow": ["error", "info", "warn"] }],
+    "no-console": ["error", { "allow": ["error", "info", "warn"] }],
```

Comment:
can we make this a "warn", to get nice green lines instead of red ones when using console.log during development?

Reply:
Sure

Reply:
with `--max-warnings 0`, we don't really need the differentiation between warning and error at all. Unfortunately, it looks like you cannot just tell the elint plugin to highlight all lint issues in green (in contrast to the tslint plugin).